src/compass/s1_geocode_slc.py

```diff
@@ -77,55 +102,73 @@ def run(cfg: GeoRunConfig):
         if cfg.rdr2geo_params.enabled:
             s1_rdr2geo.run(cfg, save_in_scratch=True)
             if cfg.rdr2geo_params.geocode_metadata_layers:
-               s1_geocode_metadata.run(cfg, fetch_from_scratch=True)
+                s1_geocode_metadata.run(cfg, burst, fetch_from_scratch=True)
```

Comment:
Apparently, the code crashes here when trying to create the HDF5 file for the metadata layers. It is not able to find the local incidence angle which is allocated in the product directory instead of the scratch directory. It would be awesome if we can fix it